src/components/Breadcrumbs/Breadcrumbs.scss

```diff
@@ -24,10 +24,6 @@ $icon-size: rem(20px);
 
   &:active {
     background-color: var(--p-surface-pressed);
-
-    .ContentWrapper {
-      background: var(--p-override-transparent);
-    }
```

Comment:
Do we need the element with `.ContentWrapper` classname any more or can we flatten this components HTML?

Reply:
Also along the line of consolidation I just want to note that there might be more css fallbacks that need to be removed in these files. I know there was work done already to remove these so it’s definitely possible that they are needed overrides. I didn’t look into them in this PR to keep a narrow scope. But if it’s something important to address now or in the future I can create a ticket to look into it.